tensorflow/lite/micro/kernels/BUILD

```diff
@@ -188,7 +188,9 @@ tflm_kernel_cc_library(
         "logical_common.cc",
         "logistic.cc",
         "logistic_common.cc",
+        "lstm_eval.cc",
         "maximum_minimum.cc",
+        "micro_tensor_utils.cc",
```

Comment:
nit: micro_tensor_utils.xx would not have target optimized version. So, it does not need to be in tflm_kernel_cc_library. It can be its own cc_library like kernel_util in this file.

Reply:
It's separated out as a standalone C++ library.